classes/userGroup/relationships/UserGroupStage.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * @file classes/userGroup/relationships/UserUserGroup.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class \PKP\userGroup\UserGroup
+ * @ingroup userGroup
```

Comment:
Now that we're using namespaces, I've been removing `@ingroup` / `@defgroup` calls.

Reply:
Removed from the newly created files in `/classes/userGroup/`